src/daemon/http/server.rs

```diff
@@ -766,7 +766,10 @@ macro_rules! aa {
                 // which causes the error message to nest, e.g.
                 //   "Invalid credentials: Invalid credentials: Session expired"
                 match err {
-                    Error::ApiInvalidCredentials(_) | Error::ApiInsufficientRights(_) => {
+                    Error::ApiInvalidCredentials(_)
```

Comment:
One oddity here is that HttpResponse::forbidden() calls self::response_from_error(ApiInsufficientRights), i.e. is the same as one of the cases in the other part of the match.